src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,18 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  (comments.length === 0) ? (
+    <>
+      <hr />
+
+      <b data-cy="NoCommentsMessage">No comments yet</b>
+    </>
+  ) : (
```

Comment:
`No comments` message shouldn't be in this component.
Render the message OR CommentList component